### DISCUSSION TITLE
Oops，这里是 404 页面。

### DISCUSSION THREAD

**Discussion by qlAD** [spencerwooo/onedrive-vercel-index#722] [Q&A]
![Screenshot_20220725-170553_Firefox](https://user-images.githubusercontent.com/54198685/180740715-6bf7e3f2-b833-4a32-820e-fb5b3fee9235.png)

**Comment by qlAD** [comment]
api确定没有问题，权限也没有问题，site设置也没有问题

**Reply by qlAD** [reply]
![Screenshot_20220725-171324_Firefox](https://user-images.githubusercontent.com/54198685/180742006-56d8be8e-c2c6-4fe5-ab05-638316011bba.png)
变量也没有问题

**Reply by Ovler-Young** [reply]
> 

https://github.com/spencerwooo/onedrive-vercel-index/discussions/569#discussioncomment-2453657

**Comment by DiLiuNEUexpresscompany** [comment]
解决了吗同样的问题

**Comment by starlight02** [comment]
如果一进入首页就显示这个提示的话，请检查下最近是否修改过 `Microsoft` 的账户密码。如有修改请删除 `upstash` 中的 redis 缓存，再刷新你的网站重新进行认证。

**Reply by Destroy-God** [reply]
怎么清除缓存啊，upstash看不到cache

**Reply by mickychenchen** [reply]
進入
https://console.upstash.com/
首頁會看到
Databases 
[onedrive-vercel-index](https://console.upstash.com/redis/)

點進去後點上方的Data Browser
把裡面的token都刪除
就可以重新認證了

**Reply by c-jeremy** [reply]
> 如果一进入首页就显示这个提示的话，请检查下最近是否修改过 `Microsoft` 的账户密码。如有修改请删除 `upstash` 中的 redis 缓存，再刷新你的网站重新进行认证。

用这一方法似乎可以成功解决此问题。

**Comment by WhaleFell** [comment]
![image](https://github.com/spencerwooo/onedrive-vercel-index/assets/63758849/42f93eed-bc03-4e6e-8335-c61e405bf58f)
原因应该是这个：ERR max daily request limit exceeded. Limit: 10000, Usage: 10000.  超出 upstash 使用限制了。

**Reply by Ovler-Young** [reply]
同样是404，也可能有不同的原因
